Core/scripts/dirac-install.py

```diff
@@ -155,6 +155,14 @@
 except ImportError:
   # Fall back to Python 2's urllib2
   from urllib2 import urlopen, HTTPError, URLError
+try:
+  long
+except NameError:
+  long = int
+try:
+  str_type = basestring
+except NameError:
+  str_type = str
```

Comment:
I understand this is for python3 (and replace what's at line 1763?). Maybe you add a comment about it (i.e. "remove this at some point")

Reply:
I changed it to use `try`/`except` to avoid the linter error that comes from `basestring` being undefined. It doesn't realise the if will never trigger but it is smart enough to realise the `NameError` will catch it. I also find having this stuff at the top is cleaner and avoids accidentally repeating the same code.

I think the comment isn't worth adding as there will be a lot of places that it applies. (Plus `dirac-install` will might need to support Python 2 until CentOS 7 end of life in 2024.)